### ISSUE TITLE
Enable verbalization of each line in embedded DartPad

### ISSUE THREAD

**Issue by atsansone** [flutter/website#10545]
From https://buganizer.corp.google.com/issues/337850112

### Observed Results

Navigating through the interactive code example, I noticed that the number of the line is not reached and verbalized by SR.

* [Screencast VoiceOver](http://screencast/cast/NTMzMTc1NzU1NjU2Mzk2OHw1OTcwYTUzYS0zZQ)
* [Screencast ChromeVox](http://screencast/cast/NTQ2NjI3MDYzMDY3NDQzMnw5YjMxN2JkZi1iNg)

### Expected Results

Ensure that when the user is navigating in every line the number also should be notified.

### User Impact

Screen reader and Cognitive users are confused because they are now aware about how many lines the code has.

### Steps to Reproduce

1. Enabled VoiceOver using <kbd>Command</kbd> + <kbd>F5</kbd> also ChromeVox using <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>Z</kbd>
1. Open the following page: https://docs.flutter.dev/cookbook/design/snackbars#interactive-example
1. Continue the linear navigation using the <kbd>Tab</kbd> and <kbd>Arrow</kbd> keys and reach the interactive example code.
1. Listen to the results inside the code of the line number.

### GAR Assessment

[Flutter Docs](https://a11ytracker.corp.google.com/products/c2fdcf18-53d2-475d-b8b0-e5dedf8cb900?e=ProductLabels)

### GAR Details

[1.2. Can all controls be reached and used via the keyboard without using a mouse?](go/GAR-web#keyboard)

### Environment Details

Machine: MAC/ChromeBook
Browser: Chrome Version 120.0.6099.272 (Official Build) (64-bit)
Screen reader: VoiceOver / ChromeVox.
Bug filling time: 15 mins.

**Comment by parlough** [comment]
Closing in favor of https://github.com/dart-lang/dart-pad/issues/1072 as the underlying issue is a DartPad issue.
